conf/rest/9.12.0/snapshotpolicy.yaml

```diff
@@ -11,10 +11,10 @@ counters:
   - ^name             => snapshot_policy
   - ^scope            => scope
 
-
 plugins:
+#  - SnapshotViolation
```

Comment:
I would have expected `SnapshotViolation` to come after SnapshotPolicy instead of before, both from a chronological and alphabetical point of view